packages/ethers-5/exports/vue.ts

```diff
@@ -0,0 +1,41 @@
+import type {
+  W3mAccountButton,
+  W3mButton,
+  W3mConnectButton,
+  W3mNetworkButton
+} from '@web3modal/scaffold'
+import type { Web3ModalOptions } from '../src/client.js'
+import { Web3Modal } from '../src/client.js'
+import { VERSION } from '@web3modal/utils'
+
+// -- Types -------------------------------------------------------------------
+export type { Web3ModalOptions } from '../src/client.js'
+
+export interface ComponentCustomProperties {
```

Comment:
I think this had to extend `declare module '@vue/runtime-core'` in order for auto complete to be available for `<w3m-` components inside vue files. Does removing it still work? I think it won't as this is also no longer a global?